app/controllers/reviews_controller.rb

```diff
@@ -8,7 +6,13 @@ def new
   def create
     shelter = Shelter.find(params[:shelter_id])
     new_review = shelter.reviews.create(review_params)
-    redirect_to "/shelters/#{shelter.id}"
+    if new_review.save
```

Comment:
Adds the flash message functionality. A message validates the entry or display an error if some of the fields have not been entered.